apps/suredbits-wallet/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   web:
-    image: bitcoinscala/wallet-server-ui:1.9.0-19b72728-SNAPSHOT@sha256:e3a4dfd7748a4cfce714beacca869c9116026c03dda9130987e151531d374290
+    image: bitcoinscala/wallet-server-ui:1.9.1@sha256:34f6f57ffd87b200895a528b37b4fb80c0d508af088753a98dc70740c0d7b93a
```

Comment:
Can you change the digest to: `sha256:3eb479b106811d523c4e0cfde244949f6c76a27c7d1fe59be9b8b51ba2372649`

This looks like the digest only for `amd64`, but the above can be used with both `amd64` and `arm64`